src/llm/llmnoderesources.cpp

```diff
@@ -104,6 +106,12 @@ void LLMNodeResources::loadTextProcessor(std::shared_ptr<LLMNodeResources>& node
     } catch (const pybind11::error_already_set& e) {
         SPDLOG_INFO(CHAT_TEMPLATE_WARNING_MESSAGE);
         SPDLOG_DEBUG("Chat template loading failed with error: {}", e.what());
+    } catch (const pybind11::cast_error& e) {
+        SPDLOG_INFO(CHAT_TEMPLATE_WARNING_MESSAGE);
+        SPDLOG_DEBUG("Chat template loading failed with error: {}", e.what());
+    } catch (const pybind11::key_error& e) {
+        SPDLOG_INFO(CHAT_TEMPLATE_WARNING_MESSAGE);
+        SPDLOG_DEBUG("Chat template loading failed with error: {}", e.what());
```

Comment:
Aren't cast and key error under error_already_set in the hierarchy? Weren't they caught before?
I wonder if we need that block if we are not changing error messages.

Reply:
They were caught in last catch block with no specified exception type. It turns out that not all errors fall into error_already_set error. Catching other types allow for easier debug since the error with wrong type (str/null) was not in the logs, now it will be @mzegla

Reply:
It changes error message - in case of this error we have `e.what()`